src/Tooltip/TooltipBubble.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import Text from '../Text';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +arrowHorizontalPosition?: 'flex-start' | 'center' | 'flex-end',
+  +arrowVerticalPosition?: 'flex-start' | 'flex-end',
+  +style?: StylePropType,
+  +text: string,
+  +verticalShift?: number,
+|};
+
+export default function TooltipBubble({
+  arrowHorizontalPosition = 'center',
+  arrowVerticalPosition = 'flex-end',
+  style,
+  text,
+  verticalShift = 3,
+}: Props) {
+  const invisibleContainerShiftStyle =
+    arrowVerticalPosition === 'flex-start'
+      ? { marginTop: verticalShift }
+      : { marginBottom: verticalShift };
+  const arrowVerticalPositionStyle =
+    arrowVerticalPosition === 'flex-start'
+      ? styles.tooltipBubbleMarginTop
+      : styles.tooltipBubbleMarginBottom;
+
+  return (
+    <View
+      style={[styles.invisibleContainer, invisibleContainerShiftStyle, style]}
+    >
+      <View
+        style={[
+          styles.arrowContainer,
+          {
+            alignItems: arrowHorizontalPosition,
+            justifyContent: arrowVerticalPosition,
+          },
+        ]}
+      >
+        <View style={styles.arrow} />
+      </View>
+      <View style={[styles.container, arrowVerticalPositionStyle]}>
+        <Text style={styles.text}>{text}</Text>
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  text: {
+    color: defaultTokens.colorTextWhite,
+    padding: parseFloat(defaultTokens.spaceXSmall),
+    zIndex: 1,
+  },
+  arrow: {
+    backgroundColor: defaultTokens.backgroundBadgeDark,
+    borderRadius: parseFloat(defaultTokens.borderRadiusSmall),
+    height: 17,
+    transform: [{ rotate: '45deg' }],
+    width: 17,
+  },
+  arrowContainer: {
+    flex: 1,
+    height: '100%',
+    paddingHorizontal: 15,
+    position: 'absolute',
+    width: '100%',
+    zIndex: 100,
```

Comment:
I believe there are also some defaultTokens for z-indexes. Would you mind having a look?

Reply:
Done